__tests__/utils/helpers.test.js

```diff
@@ -267,3 +267,9 @@ test('handlePrepareDataError', () => {
 
   expect(() => { helpers.handlePrepareDataError(err) }).toThrow(err);
 });
+
+test('cleanupString', () => {
+  expect(helpers.cleanupString('str 1')).toEqual('str 1');
+  expect(helpers.cleanupString('str  2')).toEqual('str 2');
+  expect(helpers.cleanupString('str\t 3')).toEqual('str 3');
```

Comment:
You could add spaces in the begin and in the end, and also uppercase

Reply:
Done